.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: php
+
+matrix:
+    include:
+        - php: 7.1
+        - php: 7.0
+        - php: 5.6
+        - php: 5.5
+          env: REMOVE_TWITEROAUTH=true
+        - php: 5.4
```

Comment:
The security fixes [ARE backported](https://wiki.debian.org/PHP#Notes_on_PHP_and_security). Apparently many shared hostings are running on Wheezy so there is at least some commercial interest in security of PHP 5.4.

Trust me, if I could I would drop PHP 5 altogether, it is not my call though.